libnd4j/tests_cpu/libnd4j_tests/CMakeLists.txt

```diff
@@ -19,30 +19,30 @@ if(result)
 endif()
 
 # OPTIONAL MKL-DNN
-if ("${BUILD_MKLDNN}")
+if ("${BUILD_ONEDNN}")
     # Download and unpack mkl-dnn at configure time
     configure_file(../../CMakeLists.txt.mkldnn.in mkldnn-download/CMakeLists.txt)
     execute_process(COMMAND ${CMAKE_COMMAND} -G "${CMAKE_GENERATOR}" .
         RESULT_VARIABLE result
         WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/mkldnn-download )
     if(result)
-        message(FATAL_ERROR "CMake step for mkldnn failed: ${result}")
+        message(FATAL_ERROR "CMake step for onednn failed: ${result}")
     endif()
     execute_process(COMMAND ${CMAKE_COMMAND} --build .
         RESULT_VARIABLE result
-        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/mkldnn-download )
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/onednn-download )
     if(result)
-        message(FATAL_ERROR "Build step for mkldnn failed: ${result}")
+        message(FATAL_ERROR "Build step for onednn failed: ${result}")
     endif()
 
-    add_subdirectory(${CMAKE_CURRENT_BINARY_DIR}/mkldnn-src
-        ${CMAKE_CURRENT_BINARY_DIR}/mkldnn-build
+    add_subdirectory(${CMAKE_CURRENT_BINARY_DIR}/onednn-src
+        ${CMAKE_CURRENT_BINARY_DIR}/onednn-build
         EXCLUDE_FROM_ALL)
-    set(mkldnn_SOURCE_DIR ${CMAKE_CURRENT_BINARY_DIR}/mkldnn-src)
-    set(HAVE_MKLDNN 1)
-    add_definitions("-DHAVE_MKLDNN")
-    include_directories(${mkldnn_SOURCE_DIR}/include ${mkldnn_SOURCE_DIR}/external/mklml_lnx_2019.0.3.20190220/include ${mkldnn_SOURCE_DIR})
-    set(MKLDNN dnnl)
+    set(onednn_SOURCE_DIR ${CMAKE_CURRENT_BINARY_DIR}/onednn-src)
+    set(HAVE_ONEDNN 1)
+    add_definitions("-DHAVE_ONEDNN")
+    include_directories(${onednn_SOURCE_DIR}/include ${onednn_SOURCE_DIR}/external/mklml_lnx_2019.0.3.20190220/include ${mkldnn_SOURCE_DIR})
```

Comment:
Does this use a 2019 mklml? if not, we should update the directory to reflect what it actually is.